docs/gettingStarted/Getting-Started.md

```diff
@@ -0,0 +1,122 @@
+
+### Next Steps
+
+So you decided on which codebases you want to contribute to...so...what do you do now? 
+
+The next step would be to look at the issues tab of the repository you want to contribute to. The issues tab contains a list of issues that need to be addressed, and is a good place to start if you are new to the project. You can also create your own issues if you have a feature request or a bug report. If you are new to the project, you can start by looking at the issues tagged with the `good first issue` label. These issues are relatively easy to fix, usually with minimal to little testing required. If you are looking for a more challenging issue, you can look at the issues tagged with the `help wanted` label. These issues are more challenging, and may require more effort and time, as well as testing and debugging.
+
+### Creating an Issue/Feature Request
+
+If you have a feature request or a bug report, you can create your own issue in the respective codebase repository. If you are unsure where to submit your issue or request, then choose the codebase repository where the programming language you are using is located. if using both, then choose the codebase repository where the issue is most relevant and make a note of it in the issue description.
+
+Please make sure to follow the issue template, and provide as much information as possible. If you are creating a bug report, please provide the steps to reproduce the bug, as well as the expected and actual behavior. If you are creating a feature request, please provide a detailed description of the feature, as well as the motivation behind the feature. 
+
+_Note well that the feature request may not be accepted, and may be closed if it is not in line with the project goals. Therefore, checking out the Project Roadmap [here](/project/AAI-RoadMap.md) will keep you aligned and oriented._ 
+
+### Pre-Pull Request
+
+Before you create a pull request, there are a few tasks that you should complete. First, you should make sure that your code is well documented, and that you have written unit tests for your code. You should also make sure that your code is well formatted, and that it follows the project style guide. 
+
+We provided a set of sanity check configurations which **must be passed** before creating a pull request. These configurations can be found in the `sanity-checks` folder in the respective codebase repository. You should run these configurations before creating a pull request, and make sure that they pass. If they do not pass, then you should fix the issues before creating a pull request. Additional testing/checks may be beneficial and are welcome. Finally, please state which sanity checks you have run in the pull request description and provide the output of the sanity checks.
```

Comment:
This should be mentioned in the pull request template instead of the adding tests line. We should specify what 'passing' looks like - is it sufficient to load without errors, do they have to play it etc

Reply:
Running the tests in the test folder in each codebase (named 'core') constitutes a pass. The core tests purpose is to test the fundamental functionality, and to make sure the addition/change hasnt broken anything.

Reply:
I will make a note on this in the PR template for clarification. (thanks for the feedback :))